### DISCUSSION TITLE
Support for the FeedbackUI addon.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#4818] [Suggestions]
As I was scrolling along some of the past discussions opened here, I noticed #5, which is rather clear on at least intending to support reports/feedback made through the [FeedbackUI](https://github.com/wowgaming/FeedbackUI) addon. Speaking as a contributor and someone who has been paying attention for a year and a half now, it's not hard to see that the barrier of entry of making a Github account and opening an issue *is* actually too great for some people, mostly due to a confusing user interface for newcomers and the need for an account to be made just to report bugs for a private server. There have likely been times where I myself have observed an issue in-game and put off the steps required to fill out an issue report long enough to forget about the issue, entirely due to the fact that it requires opening up the AC/CC issues page and, well, filling out the report there. It's hard to ignore the convenience of being able to make reports/submit feedback directly in-game immediately after doing the content (quest, fight, item, spell, what have you).

**Comment by gettheboom** [comment]
This is the first time when I've seen the FeedbackUI addon, but I do support the idea of having easier access to report a quest bug for example.

+
